src/Components/QuestionList/QuestionCard/QuestionCardS.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../Sass/main.scss";
+
+.listBtn {
+    margin: 1rem;
+}
+
+.listItem.card {
```

Comment:
This globally sets this class. Typically we want to nest things into component name css classNames:
```
.QuestionCard {
  .listItem.card {
    ...etc...
  }
}
```

Reply:
Same for `.listBtn` above

Reply:
fixed